practice_app/api/urls.py

```diff
@@ -1,5 +1,10 @@
 from django.urls import path
 
-urlpatterns = [
+from . import views
 
-]
\ No newline at end of file
+app_name = "api"
+urlpatterns = [
+    path('eric/', views.eric_index, name='eric_index'),
```

Comment:
I think you should omit the index page and just provide the GET function in this url.

Reply:
I omitted the index page. The GET function can be utilized from this 'eric/' endpoint.